fluidkeys/secretsend.go

```diff
@@ -27,14 +27,31 @@ import (
 
 	"github.com/fluidkeys/crypto/openpgp"
 	"github.com/fluidkeys/crypto/openpgp/armor"
+	"github.com/fluidkeys/fluidkeys/api"
+	"github.com/fluidkeys/fluidkeys/colour"
 	"github.com/fluidkeys/fluidkeys/out"
 	"github.com/fluidkeys/fluidkeys/pgpkey"
 )
 
 func secretSend(recipientEmail string) exitCode {
 	armoredPublicKey, err := client.GetPublicKey(recipientEmail)
 	if err != nil {
-		printFailed("Couldn't get the public key for " + recipientEmail + "\n")
+		if err == api.ErrPublicKeyNotFound {
+			out.Print("\n")
+			out.Print("Couldn't find " + recipientEmail + " in the Fluidkeys Directory.\n\n")
+			out.Print("You can invite them to install Fluidkeys:\n")
+			out.Print("───\n")
+			out.Print(colour.Warning(`I'd like to send you an encrypted secret with Fluidkeys.
+
+You can download and set up Fluidkeys here:
+
+https://getstarted.fluidkeys.com#` + recipientEmail + `
```

Comment:
We don't currently have `getstarted.fluidkeys.com`, nor do we have any JS that actually manipulates the instructions. 

ATM `https://download.fluidkeys.com#test`redirects to `https://www.fluidkeys.com/download/#test` so we could implement that jsmagic there?

So replace `getstarted.` with `download.` ?